test/cases/api.sh

```diff
@@ -613,6 +665,46 @@ function _instanceWaitSSH() {
   done
 }
 
+function _instanceCheck() {
+  echo "✔️ Instance checking"
+  local _ssh="$1"
+
+  # Check if postgres is installed
+  $_ssh rpm -q postgresql
+
+  pretty_name=$($_ssh grep '^PRETTY_NAME=' /etc/os-release)
+  # Beta image doesn't have available subscription so skip subscription check. See COMPOSER-996
+  if [[ "$ID" =~ "rhel" ]] && [[ ! "$pretty_name" =~ 'Beta' ]]; then
```

Comment:
see my comment above and check only if SUBSCRIPTION_BLOCK isn't empty. Then we'll have the selection logic in one place above and this function doesn't need to know on what OS variant/version it is running.